ceph_iscsi_config/lun.py

```diff
@@ -67,14 +71,20 @@ def create(self):
 
         with rados.Rados(conffile=settings.config.cephconf,
                          name=settings.config.cluster_client_name) as cluster:
-            with cluster.open_ioctx(self.pool) as ioctx:
+            _pool = self.pool
+            data_pool = None
+            if self.is_erasure_pool:
+                _pool = settings.config.pool
+                data_pool = self.pool
+            with cluster.open_ioctx(_pool) as ioctx:
```

Comment:
I'm not sure doing it this way is a good idea.  Constraining the metadata pool to `SYS_SETTINGS.pool` seems unnecessarily restrictive.  Why not add an optional `data_pool` parameter to the create RPC, allowing the user to specify both pools?

Reply:
Yeah, this also looks good to me.